SUSE/rmt
# Add how to restart the rmt-server when /etc/rmt.conf was altered

## Description

Adding a description how to restart the `rmt-server` when `/etc/rmt.conf` was changed.

Fixes # (issue)
https://github.com/SUSE/rmt/issues/709

## Change Type

*Please select the correct option.*


## Checklist

*Please check off each item if the requirement is met.*


## How to review

- Build the package
- Install rmt-server and run `man rmt-cli` and check the added line
- Run the command to restart the `rmt-server`
